example/Python/src/Piano/README.md

```diff
@@ -0,0 +1,21 @@
+# Piano
+This is a pygame-based piano demo for the Python target of Lingua Franca. 
+The contents of the demo is mostly adapted from: https://github.com/bspaans/python-mingus/tree/master/mingus_examples/pygame-piano
+
+## Installation
+### MacOs:
+```bash
+$ brew install sdl2 sdl2_gfx sdl2_image sdl2_mixer sdl2_net sdl2_ttf
+$ brew install Caskroom/cask/xquartz
+$ brew install fluid-synth
+$ python3 -m pip install mingus
+$ python3 -m pip install pygame
+```
+Then, go to ```System Preferences -> Security & Privacy -> Privacy -> Input Monitoring``` and check Terminal so that the Terminal application can read keystrokes.
+
+### Linux:
+```
```

Comment:
```suggestion
```bash
 $ sudo apt install libsdl2-2* libsdl2-image-2*
 $ sudo apt install fluidsynth
 $ pip3 install mingus
 $ pip3 install pygame
```

Reply:
I've added the `libsdl` and `fluidsynth` requirements to @lhstrh's suggestion. This resulted in a working demo on my Ubuntu Linux machine with [this](https://github.com/lf-lang/lingua-franca/pull/565#issuecomment-933153502) fix.

Reply:
the ```if not fluidsynth.init(SF2, "alsa"):``` line from ```sensor_simulator.py``` basically calls functions from [this file](https://github.com/bspaans/python-mingus/blob/master/mingus/midi/fluidsynth.py).

```Python
 def start_audio_output(self, driver=None):
        """Start the audio output.
        The optional driver argument can be any of 'alsa', 'oss', 'jack',
        'portaudio', 'sndmgr', 'coreaudio', 'Direct Sound', 'dsound',
        'pulseaudio'. Not all drivers will be available for every platform.
        """
        self.fs.start(driver)
```
from the comments in the file, it seems the "alsa" fix is indeed platform dependent.